sdk/scheduler/src/main/java/com/mesosphere/sdk/scheduler/DefaultScheduler.java

```diff
@@ -58,47 +45,36 @@
 import java.util.concurrent.ExecutorService;
 import java.util.concurrent.Executors;
 import java.util.concurrent.TimeUnit;
-import java.util.concurrent.atomic.AtomicBoolean;
 import java.util.stream.Collectors;
 
 /**
  * This scheduler when provided with a ServiceSpec will deploy the service and recover from encountered faults
  * when possible.  Changes to the ServiceSpec will result in rolling configuration updates, or the creation of
  * new Tasks where applicable.
  */
-public class DefaultScheduler implements Scheduler, Observer {
+public class DefaultScheduler extends AbstractScheduler implements Observer {
 
     /**
      * Time to wait for the executor thread to terminate. Only used by unit tests.
```

Comment:
Ah. I should have appended my comment one line lower. The method went from protected -> private.